manage.py

```diff
@@ -3,7 +3,7 @@
 import sys
 
 if __name__ == "__main__":
-    os.environ.setdefault("DJANGO_SETTINGS_MODULE", "config.settings.production")
+    os.environ.setdefault("DJANGO_SETTINGS_MODULE", "config.settings.local")
```

Comment:
well, it was set to production because we will deploy using pipeline, no manual commands
you can manually set DJANGO_SETTINGS_MODULE ENV on your localhost using the export command .